jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -247,45 +257,45 @@ def _get_2d_pixgrid(model2d, channel):
 
     y, x = np.indices((1024, 512))
 
-    if channel in ['1', '4']:
+    if channel in ["1", "4"]:
         return x + 4, y
     else:
         return x + 516, y
 
 
-def _match_models(models, channel, degree, center=None, center_cs='image'):
-    from .. cube_build import CubeBuildStep
+def _match_models(models, channel, degree, center=None, center_cs="image"):
+    from ..cube_build import CubeBuildStep
```

Comment:
why is this import embedded in the function?

Reply:
I have a basic understanding of how this step works. Early in mission we used it. This step calls cube_build in 'single' mode cube to determine a background. This was done per channel/band for MRS data. The idea is that the background could have a variation (one source telescope background differences at the longer wavelengths of the MRS). This step would determine polynomial to subtract to remove any variations but by default it would not subtract the background. It was left to cube_build to subtract it if the polynomial was in the meta data of the model.